texttmining.py

```diff
@@ -0,0 +1,106 @@
+import requests
+import nltk
+import pickle
+from bs4 import BeautifulSoup
+
+#Mary_Wade_text = requests.get('http://www.gutenberg.org/cache/epub/43585/pg43585.txt').text
+#print(Mildred_Wirt_text)
+
+# Save data to a file (will be part of your data fetching script)
+# f = open('Mary_Wade_text.pickle', 'wb')
+# pickle.dump(Mary_Wade_text, f)
+# f.close()
+
```

Comment:
It is a good practice to delete commented out, unnecessary chunk of code that is not part of your final program.